src/components/dashboard/dashboard.directive.coffee

```diff
@@ -10,7 +10,11 @@ module.controller('ImpacDashboardCtrl', ($scope, $http, $q, $filter, $modal, $lo
     # references to services (bound objects shared between all controllers)
     # -------------------------------------
     $scope.currentDhb = ImpacDashboardsSvc.getCurrentDashboard()
-    $scope.widgetsList = ImpacDashboardsSvc.getWidgetsTemplates()
+    # developers can add widgets via the ImpacDeveloper service without modifying the API
+    if ImpacDeveloper.getStatus()
```

Comment:
So, only the stubbed templates will populate scope.widgetsList?
What happens if the developer loads an existing dashboard with widgets which won't find their templates?

Couldn't we merge the stubbed one into the full list?